commit 14290db2571d3d1b71e1f3b0f2549ac03a9141a5
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-07-24T10:24:46-07:00 (3 months ago)
    
    TRITON-1835 cloudapi migration tests do not need to check the sync message field

diff --git a/test/machines/migrations.js b/test/machines/migrations.js
index 84fb0004c..9239dd9fc 100644
--- a/test/machines/migrations.js
+++ b/test/machines/migrations.js
@@ -194,7 +194,6 @@ module.exports = function migrate(suite, client, other, machine, callback) {
                 });
                 progressEvents.forEach(function testPrgrEvt(evt) {
                     t.ok(evt.current_progress, 'Should have current_progress');
-                    t.ok(evt.message, 'Should have a message');
                     t.ok(evt.state, 'Should have event state');
                     t.ok(evt.total_progress, 'Should have total_progress');
                     if (evt.started_timestamp) {
